dbus-update-activation-environment: setting DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
dbus-update-activation-environment: setting DISPLAY=:0
dbus-update-activation-environment: setting XAUTHORITY=/home/sammy/.Xauthority
/etc/X11/Xsession.d/30x11-common_xresources: line 16: has_option: command not found
/etc/X11/Xsession.d/75dbus_dbus-launch: line 9: has_option: command not found
dbus-update-activation-environment: setting GTK_MODULES=gail:atk-bridge
dbus-update-activation-environment: setting QT_ACCESSIBILITY=1
/etc/X11/Xsession.d/90x11-common_ssh-agent: line 9: has_option: command not found
dbus-update-activation-environment: setting SHELL=/bin/bash
dbus-update-activation-environment: setting QT_ACCESSIBILITY=1
dbus-update-activation-environment: setting XDG_CONFIG_DIRS=/etc/xdg/xdg-cinnamon:/etc/xdg
dbus-update-activation-environment: setting LANGUAGE=en_IN:en
dbus-update-activation-environment: setting LC_ADDRESS=en_IN
dbus-update-activation-environment: setting LC_NAME=en_IN
dbus-update-activation-environment: setting DESKTOP_SESSION=cinnamon
dbus-update-activation-environment: setting LC_MONETARY=en_IN
dbus-update-activation-environment: setting GTK_MODULES=gail:atk-bridge
dbus-update-activation-environment: setting PWD=/home/sammy
dbus-update-activation-environment: setting XDG_SESSION_DESKTOP=cinnamon
dbus-update-activation-environment: setting LOGNAME=sammy
dbus-update-activation-environment: setting XDG_SESSION_TYPE=x11
dbus-update-activation-environment: setting GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1
dbus-update-activation-environment: setting XAUTHORITY=/home/sammy/.Xauthority
dbus-update-activation-environment: setting XDG_GREETER_DATA_DIR=/var/lib/lightdm-data/sammy
dbus-update-activation-environment: setting GDM_LANG=en_US
dbus-update-activation-environment: setting HOME=/home/sammy
dbus-update-activation-environment: setting IM_CONFIG_PHASE=1
dbus-update-activation-environment: setting LANG=en_IN
dbus-update-activation-environment: setting LC_PAPER=en_IN
dbus-update-activation-environment: setting XDG_SESSION_CLASS=user
dbus-update-activation-environment: setting LC_IDENTIFICATION=en_IN
dbus-update-activation-environment: setting USER=sammy
dbus-update-activation-environment: setting DISPLAY=:0
dbus-update-activation-environment: setting SHLVL=0
dbus-update-activation-environment: setting LC_TELEPHONE=en_IN
dbus-update-activation-environment: setting LC_MEASUREMENT=en_IN
dbus-update-activation-environment: setting XDG_RUNTIME_DIR=/run/user/1000
dbus-update-activation-environment: setting LC_TIME=en_IN
dbus-update-activation-environment: setting GTK3_MODULES=xapp-gtk3-module
dbus-update-activation-environment: setting XDG_DATA_DIRS=/usr/share/cinnamon:/usr/share/gnome:/home/sammy/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share:/usr/share
dbus-update-activation-environment: setting PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
dbus-update-activation-environment: setting GDMSESSION=cinnamon
dbus-update-activation-environment: setting DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
dbus-update-activation-environment: setting LC_NUMERIC=en_IN
dbus-update-activation-environment: setting _=/usr/bin/dbus-update-activation-environment
discover_other_daemon: 1discover_other_daemon: 1discover_other_daemon: 1[cinnamon-settings-daemon-smartcard] Failed to start: no suitable smartcard driver could be found
dbus-daemon[1985]: Activating service name='org.a11y.atspi.Registry' requested by ':1.6' (uid=1000 pid=1991 comm="/usr/bin/csd-clipboard" label="unconfined")
dbus-daemon[1985]: Successfully activated service 'org.a11y.atspi.Registry'
[cinnamon-settings-daemon-smartcard] Failed to start: no suitable smartcard driver could be found
/usr/bin/cinnamon-launcher:56: DeprecationWarning: This process (pid=2122) is multi-threaded, use of fork() may lead to deadlocks in the child.
  self.cinnamon_pid = os.fork()
Gjs-Message: 14:57:44.957: Profiler is disabled. Not setting up signals.
Gjs-Message: 14:57:45.080: JS LOG: About to start Cinnamon (X11 backend)
Gjs-Message: 14:57:45.137: JS LOG: [LookingGlass/info] Cinnamon.AppSystem.get_default() started in 0 ms
Gjs-Message: 14:57:45.139: JS LOG: [LookingGlass/info] loading user theme: /usr/share/themes/Mint-Y-Dark-Purple/cinnamon/cinnamon.css
Gjs-Message: 14:57:45.150: JS LOG: [LookingGlass/info] added icon directory: /usr/share/themes/Mint-Y-Dark-Purple/cinnamon
Gjs-Message: 14:57:45.226: JS LOG: [LookingGlass/info] loaded at Sat Jun 07 2025 14:57:45 GMT+0530 (India Standard Time)
Gjs-Message: 14:57:45.226: JS LOG: Cinnamon started at Sat Jun 07 2025 14:57:45 GMT+0530 (India Standard Time)
Gjs-Message: 14:57:45.234: JS LOG: [LookingGlass/info] ExtensionSystem started in 0 ms
Gjs-Message: 14:57:45.234: JS LOG: [LookingGlass/info] DeskletManager started in 0 ms
Gjs-Message: 14:57:45.234: JS LOG: [LookingGlass/info] SearchProviderManager started in 0 ms
Gjs-Message: 14:57:45.268: JS LOG: [LookingGlass/info] Loaded applet removable-drives@cinnamon.org in 12 ms
Gjs-Message: 14:57:45.270: JS LOG: [LookingGlass/info] Loaded applet separator@cinnamon.org in 2 ms
Gjs-Message: 14:57:45.272: JS LOG: [LookingGlass/info] Role locked: tray
Gjs-Message: 14:57:45.273: JS LOG: [LookingGlass/info] Loaded applet systray@cinnamon.org in 3 ms

** (at-spi2-registryd:2060): WARNING **: 14:57:45.338: Failed to register client: GDBus.Error:org.gnome.SessionManager.AlreadyRegistered: Unable to register client

** (at-spi2-registryd:2060): WARNING **: 14:57:45.338: Unable to register client with session manager
Gjs-Message: 14:57:45.358: JS LOG: [LookingGlass/info] Loaded applet printers@cinnamon.org in 85 ms
Gjs-Message: 14:57:45.372: JS LOG: [LookingGlass/info] Role locked: notifications
Gjs-Message: 14:57:45.382: JS LOG: [LookingGlass/info] Loaded applet notifications@cinnamon.org in 23 ms
Gjs-Message: 14:57:45.405: JS LOG: [LookingGlass/info] Loaded applet keyboard@cinnamon.org in 23 ms
Gjs-Message: 14:57:45.441: JS LOG: [LookingGlass/info] Loaded applet cornerbar@cinnamon.org in 36 ms
Gjs-Message: 14:57:45.467: JS LOG: [LookingGlass/info] Loaded applet favorites@cinnamon.org in 26 ms
Gjs-Message: 14:57:45.473: JS LOG: [LookingGlass/info] Loaded applet xapp-status@cinnamon.org in 5 ms
Gjs-Message: 14:57:45.530: JS LOG: [LookingGlass/info] Loaded applet calendar@cinnamon.org in 57 ms
Gjs-Message: 14:57:45.563: JS LOG: [LookingGlass/info] Role locked: panellauncher
Gjs-Message: 14:57:45.563: JS LOG: [LookingGlass/info] Role locked: windowattentionhandler
Gjs-Message: 14:57:45.567: JS LOG: [LookingGlass/info] Loaded applet grouped-window-list@cinnamon.org in 37 ms
Gjs-Message: 14:57:45.581: JS LOG: [LookingGlass/info] Loaded applet power@cinnamon.org in 14 ms
Gjs-Message: 14:57:45.632: JS LOG: [LookingGlass/info] Adding XAppStatusIcon: nm-applet (:1.53/org/x/StatusIcon/Icon)

ERROR: NVIDIA driver is not loaded


** (csd-power:1973): CRITICAL **: 14:57:45.660: abs_to_percentage: assertion 'max > min' failed
Gjs-Message: 14:57:45.677: JS LOG: [LookingGlass/info] Loaded applet sound@cinnamon.org in 96 ms
Nemo-Message: 14:57:45.696: nemo-desktop: session is cinnamon, establishing proxy
Gjs-Message: 14:57:45.704: JS LOG: [LookingGlass/info] Hiding XAppStatusIcon (we have an applet): nm-applet (:1.53/org/x/StatusIcon/Icon)
Gjs-Message: 14:57:45.711: JS LOG: [LookingGlass/info] Loaded applet network@cinnamon.org in 34 ms
blueman-applet 14.57.45 WARNING  PluginManager:150 __load_plugin: Not loading PPPSupport because its conflict has higher priority
blueman-applet 14.57.45 WARNING  PluginManager:150 __load_plugin: Not loading DhcpClient because its conflict has higher priority
Gjs-Message: 14:57:45.798: JS LOG: [LookingGlass/info] Loaded applet menu@cinnamon.org in 87 ms
Gjs-Message: 14:57:45.799: JS LOG: [LookingGlass/info] AppletManager started in 573 ms
Gjs-Message: 14:57:45.802: JS LOG: [LookingGlass/info] Cinnamon took 722 ms to start
Gjs-Message: 14:57:45.821: JS LOG: calendar@cinnamon.org: Calendar events supported.
Gjs-Message: 14:57:45.847: JS LOG: Unknown network device type, is 32
Gjs-Message: 14:57:45.935: JS LOG: [LookingGlass/info] GPU offload supported: false
Gjs-Message: 14:57:45.942: JS LOG: [LookingGlass/info] Adding XAppStatusIcon: blueman (:1.50/org/x/StatusIcon/Icon)
Gjs-Message: 14:57:46.652: JS LOG: [LookingGlass/info] Adding XAppStatusIcon: nvidia-prime (:1.57/org/x/StatusIcon/Icon)
Nemo-Share-Message: 14:57:46.945: Called "net usershare info" but it failed: 'net usershare' returned error 255: net usershare: cannot open usershare directory /var/lib/samba/usershares. Error No such file or directory
Please ask your system administrator to enable user sharing.

Gjs-Message: 14:57:49.150: JS LOG: Enabling WindowAttentionHandler
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:1394: DeprecationWarning: Gdk.threads_init is deprecated
  Gdk.threads_init()
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:1778: DeprecationWarning: Gdk.threads_enter is deprecated
  Gdk.threads_enter()
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:817: DeprecationWarning: Gdk.threads_leave is deprecated
  Gdk.threads_leave()
Gjs-Message: 14:58:05.712: JS LOG: [LookingGlass/info] Adding XAppStatusIcon: mintUpdate.py (:1.90/org/x/StatusIcon/Icon)
Gjs-Message: 14:58:25.308: JS LOG: [LookingGlass/info] Adding XAppStatusIcon: mintreport (:1.94/org/x/StatusIcon/Icon)
Gjs-Message: 14:58:25.311: JS LOG: [LookingGlass/info] Adding XAppStatusIcon: process-monitor (:1.94/org/x/StatusIcon/Icon_1)
/usr/lib/python3/dist-packages/apt/cache.py:562: Warning: W:An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://repo.protonvpn.com/debian stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Failed to fetch https://repo.protonvpn.com/debian/dists/stable/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Some index files failed to download. They have been ignored, or old ones used instead.
  res = self._cache.update(fetch_progress, slist, pulse_interval)
/usr/lib/python3/dist-packages/apt/cache.py:562: Warning: W:An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://repo.protonvpn.com/debian stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Failed to fetch https://repo.protonvpn.com/debian/dists/stable/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Some index files failed to download. They have been ignored, or old ones used instead.
  res = self._cache.update(fetch_progress, slist, pulse_interval)
/usr/lib/python3/dist-packages/apt/cache.py:562: Warning: W:An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://repo.protonvpn.com/debian stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Failed to fetch https://repo.protonvpn.com/debian/dists/stable/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Some index files failed to download. They have been ignored, or old ones used instead.
  res = self._cache.update(fetch_progress, slist, pulse_interval)
/usr/lib/python3/dist-packages/apt/cache.py:562: Warning: W:An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://repo.protonvpn.com/debian stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Failed to fetch https://repo.protonvpn.com/debian/dists/stable/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Some index files failed to download. They have been ignored, or old ones used instead.
  res = self._cache.update(fetch_progress, slist, pulse_interval)
/usr/lib/python3/dist-packages/apt/cache.py:562: Warning: W:An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://repo.protonvpn.com/debian stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Failed to fetch https://repo.protonvpn.com/debian/dists/stable/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EDA3E22630349F1C, W:Some index files failed to download. They have been ignored, or old ones used instead.
  res = self._cache.update(fetch_progress, slist, pulse_interval)
